sapi/fpm/fpm/fpm_sockets.c

```diff
@@ -67,7 +83,10 @@ static void fpm_sockets_cleanup(int which, void *arg) /* {{{ */
 	}
 
 	if (env_value) {
-		setenv("FPM_SOCKETS", env_value, 1);
+		for(i = 0; i < socket_set_count; i++) {
+			sprintf(envname, "FPM_SOCKETS_%d", i);
```

Comment:
I would probably keep the one with 0 index without the suffix (it means keeping `FPM_SOCKETS` for the first one). The reason is that technically this env can be accessible in the script (at least I think it can be if `clear_env = off`). Although I don't think there would be any use for this env, I don't want to remove it as this should probably go to 7.1 as well which is quite stable.